dm/tests/full_mode/run.sh

```diff
@@ -202,6 +202,10 @@ function run() {
 	# start DM task only
 	dmctl_start_task "$cur/conf/dm-task.yaml" "--remove-meta"
 
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"stage\": \"Finished\"" 2
```

Comment:
run_sql_file is blocking, why would it be in the middle state

Reply:
i didn't recurrence this [#5490](https://github.com/pingcap/tiflow/issues/5490) when ran locally too, but there was diffrent time between check-diff and clean log in worker2, so i guess it may be in a middle state, maybe the data is coincident. but data files weren't deleted. btw, other case in full_mode has wait `query-status Finished`.

Reply:
`check_sync_diff` failed in CI job because it task runs slow?

if it's that way, it may not work, since `run_dm_ctl_with_retry` wait for `10*2` seconds, but `check_sync_diff` waits `20*2` seconds

Reply:
no, `check_sync_diff` pass, but check dump files removed failed.
https://github.com/pingcap/tiflow/pull/5507/files#diff-e1e9a23978ee6357d40677cb4b58be9c20f004503623b09526009f2f7c7e7ae0L210